enums/FeatureType.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+
+from CustomExceptions.UnsupportedOperationException import UnsupportedOperationException
```

Comment:
you can pack CustomExceptions in ___init___.py, so your import will like this:
from exceptions import UnsupportedOperationException